### PR TITLE
Rename .marketplace-kit to .pos

### PR DESCRIPTION
**Pull request by pavelloz**
This PR deprecates `.marketplace-kit` file in favor of `.pos`.

It looks for config in this order:
1. `CONFIG_FILE_PATH` env
2. `.pos`
3. `.marketplace-kit`

When adding new env it creates .pos.

It should work with all three variants, and if all files exist, it will respect the order above when adding new one.

QA steps:
* Adding & listing envs should work as usual
* Deploys should work
* Sync should work
* Listing modules & removing modules should work

### CODE REVIEW

File: bin/pos-cli-data-clean.js [hunk 1]
```diff
@@ -33,17 +33,14 @@ const confirmCleanup = async (gateway, inlineConfirmation) => {
     clean(gateway);
   } else {
     logger.Error('Wrong confirmation. Closed without cleaning instance data.');
-    process.exit(1);
```
Comment (piotrze) [inline bin/pos-cli-data-clean.js]: @pavelloz But why?

Comment (pavelloz) [reply bin/pos-cli-data-clean.js]: https://github.com/mdyd-dev/pos-cli/blob/dd3e7fd539aa2a6c7f903a458067ef9b2c85f88a/lib/logger.js#L25

Why not?

Comment (Developer-DNM) [reply bin/pos-cli-data-clean.js]: You guys make me lol sometimes.

**Comment by pavelloz** [comment]
From pos-cli POV, if there is no error and worked once, its somewhere else.

**Pull request merged**